fbw-a32nx/src/systems/fmgc/src/flightplanning/AircraftConfigTypes.ts

```diff
@@ -70,6 +49,16 @@ export interface VnavConfig {
    * Configurable since different Airbus aircraft use different labels (e.g. A320 vs A380).
    */
   LIM_PSEUDO_WPT_LABEL: '(LIM)' | '(SPDLIM)';
+
+  /**
+   * The maximum operating speed in knots
+   */
+  VMO: number;
+
+  /**
+   * The maximum operating Mach number
+   */
+  MMO: number;
```

Comment:
```suggestion
  /**
   * The maximum operating speed in knots
   */
  Vmo: number;

  /**
   * The maximum operating Mach number
   */
  Mmo: number;
```
Our current style.

Reply:
Do we have this convention written down somewhere? I also prefer PascalCase for these parameters that do not change at runtime, but it does not appear that this has been enforced so far.

Reply:
We should probably write them down, yeah.

Reply:
I have opened a separate PR to convert them https://github.com/flybywiresim/aircraft/pull/9865. It would be ideal to have a lint rule for this, but I have not found one that does this.